Source/Core/Core/IOS/ES/Formats.cpp

```diff
@@ -138,6 +142,19 @@ u16 TMDReader::GetGroupId() const
   return Common::swap16(m_bytes.data() + offsetof(TMDHeader, group_id));
 }
 
+std::string TMDReader::GetGameID() const
+{
+  char game_id[6];
+  std::memcpy(game_id, m_bytes.data() + offsetof(TMDHeader, title_id) + 4, 4);
```

Comment:
`sizeof(TMDHeader::title_id)` and `sizeof(TMDHeader::group_id)` (or `sizeof(u32)` and `sizeof(u16)`) can be used instead of magic constants here.